cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/BasicPushStrategy.java

```diff
@@ -221,7 +219,7 @@ private Changes getNextBatch() throws ExecutionException, InterruptedException ,
         long lastPushSequence = getLastCheckpointSequence();
         logger.fine("Last push sequence from remote database: " + lastPushSequence);
         return this.sourceDb.getDbCore().changes(lastPushSequence,
-                config.changeLimitPerBatch);
+                this.changeLimitPerBatch);
```

Comment:
Also not part of your original changes in this PR but it looks like there should be a space before the opening bracket for this method block and a space should be removed after the InterruptedException reference.  Line [218](https://github.com/cloudant/sync-android/pull/222/files#diff-5d839a25cecd37453f072e5eb2df6cf4R218) should look like:

```java
private Changes getNextBatch() throws ExecutionException, InterruptedException, DatastoreException {
...
```